### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by linnanwang** [pytorch/botorch#918]
# 🐛 Bug

I know I used it a wrong way, but the output will have some scalars > 1.

## To reproduce

** Code snippet to reproduce **

import torch
import torch.nn as nn
import torch.nn.functional as F
import random
import copy as cp
import sys
import os
import pickle

soboleng = torch.quasirandom.SobolEngine(dimension=20, seed = 10)

counter = 0
for j in range(0, 1000000):
    soboleng.fast_forward(counter)
    samples = soboleng.draw(10)
    for i in range(0, samples.shape[0]):
        if samples[i][-1] > 1:
            print(samples)
    counter += 10

## Expected Behavior

all scalars should within [0, 1].

**Comment by Balandat** [comment]
Ugh `7.8102e+09` - there is some overflow going on somewhere.

Could you please open an issue in https://github.com/pytorch/pytorch/issues since this is a pytorch issue? Thanks!

**Comment by Balandat** [comment]
I was taking another look at this and I'm unable to repro this, either with this seed or other seeds. Maybe this was an more general issue with PyTorch unrelated to the Sobol generation that has since been fixed?

I'm going to close this for now, if this still persists please reopen and provide details on the environment you're running this in.
